packages/datadog-plugin-winston/src/index.js

```diff
@@ -15,7 +15,8 @@ function chunkProxy (chunk, holder) {
       }
     },
     ownKeys (target) {
-      return ['dd', ...Reflect.ownKeys(target)]
+      const ownKeys = Reflect.ownKeys(target)
+      return ownKeys.includes('dd') ? ownKeys : ['dd', ...ownKeys]
```

Comment:
could we do a Object.has / hasOwnProperty for 'dd' instead of iterating the array here?

Reply:
Yes. Will do.